build.gradle

```diff
@@ -37,20 +17,44 @@ buildscript {
         }
         classpath 'com.github.rholder:gradle-one-jar:1.0.3'
         classpath 'org.ow2.proactive:gradle-serialver:2.2'
-        classpath 'org.postgresql:postgresql:9.3-1102-jdbc41'  
+        classpath 'org.postgresql:postgresql:9.3-1102-jdbc41'
         classpath 'org.ajoberstar:gradle-jacoco:0.3.0'
         classpath('org.gradle.api.plugins:gradle-format-plugin:1.1') {
-                    exclude group: 'javassist', module: 'javassist'
+            exclude group: 'javassist', module: 'javassist'
         }
 
         classpath 'com.qmino:miredot-plugin:1.6.2'
     }
 }
 
+plugins {
```

Comment:
It seems not possible:

https://plugins.gradle.org/plugin/org.sonarqube#new-plugin-mechanism-info-body

Please ignore my comment.